proto/storage/base_image_repository.proto

```diff
@@ -0,0 +1,71 @@
+syntax = "proto3";
+
+package storage;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "./storage;storage";
+option java_package = "io.stackrox.proto.storage";
+
+// BaseImageRepository configures automatic tag discovery for a base image repository.
+// Defines which repository to watch, what tag patterns to match, and health tracking.
+// Next tag: 8
+message BaseImageRepository {
+  // Unique identifier for this repository pattern configuration.
+  string id = 1; // @gotags: sql:"pk,type(uuid)"
+
+  // Full repository path including registry (e.g., "registry.redhat.io/ubi8/ubi").
+  // Must be unique - one tag pattern per repository.
+  string repository_path = 2; // @gotags: sql:"unique"
+
+  // Glob pattern for tag filtering (e.g., "8.10-*").
+  string tag_pattern = 3;
+
+  // Timestamp of last successful poll.
+  google.protobuf.Timestamp last_poll_at = 4;
+
+  // Count of consecutive poll failures (for health tracking).
+  int32 failure_count = 5;
+
+  // Health status of this repository pattern.
+  enum HealthStatus {
+    HEALTHY = 0; // Pattern polling successfully.
+    UNHEALTHY = 1; // Pattern experiencing failures.
+    DISABLED = 2; // Pattern disabled by operator.
+  }
+  HealthStatus health_status = 6; // @gotags: sql:"index=btree"
+
+  // SHA256 hash of tag_pattern for cache invalidation detection
+  // (when pattern changes, hash differs, triggering cache flush).
+  string pattern_hash = 7;
+}
+
+// BaseImageTag stores cached base image tag metadata.
+// Persisting cache data across Central restarts avoids cold-start refetches during base
+// image repository scanning.
+// Next tag: 8
+message BaseImageTag {
+  // Unique identifier.
+  string id = 1; // @gotags: sql:"pk,type(uuid)"
+
+  // Foreign key to the repository pattern.
+  // Composite unique constraint with tag field.
+  string base_image_repository_id = 2; // @gotags: sql:"fk(BaseImageRepository:id),type(uuid),index=category:unique;name:base_image_tags_repo_tag"
+
+  // Tag name (e.g., "8.10-1234").
+  // Composite unique constraint with base_image_repository_id field.
+  string tag = 3; // @gotags: search:"Base Image Tag,hidden" sql:"index=category:unique;name:base_image_tags_repo_tag"
+
+  // SHA256 digest of the manifest (or manifest list for multi-arch).
+  string manifest_digest = 4;
+
+  // Image creation timestamp from config blob.
+  google.protobuf.Timestamp created = 5;
+
+  // Whether this tag points to a manifest list (multi-platform).
+  bool is_manifest_list = 6;
```

Comment:
It would be better to move this to be together with `manifest_digest` at line 60. It looks like they should be together.

Reply:
For readability? Or is it something else?